include/tvm/relay/attrs/transform.h

```diff
@@ -321,6 +321,12 @@ struct ArgWhereAttrs : public tvm::AttrsNode<ArgWhereAttrs> {
   }
 };  // struct ArgWhereAttrs
 
+/*! \brief Attributes used in unravel_index operators */
+struct UnRavelIndexAttrs : public tvm::AttrsNode<UnRavelIndexAttrs> {
+  TVM_DECLARE_ATTRS(UnRavelIndexAttrs, "relay.attrs.UnRavelIndexAttrs") {
```

Comment:
I don't think there's any need to define an attribute type for an operator without attributes. Although `argwhere` seems to do the same thing you have, other operators without attributes just don't use one (see `nn.batch_flatten` as one example). I'd argue we should try to avoid defining unnecessary attrs to prevent bloat.

Reply:
Ok. Thanks. This is good to know. I have removed the attrs for both unravel_index and argwhere.